kotest-framework/kotest-framework-engine/src/jvmTest/kotlin/com/sksamuel/kotest/engine/test/interceptors/CoroutineDispatcherInterceptorTest.kt

```diff
@@ -32,9 +32,8 @@ class CoroutineDispatcherInterceptorTest : DescribeSpec() {
             )
 
             val controller = object : CoroutineDispatcherFactory {
-               @OptIn(DelicateCoroutinesApi::class)
                override suspend fun <T> withDispatcher(testCase: TestCase, f: suspend () -> T): T =
-                  newSingleThreadContext("foo").use { dispatcher ->
+                  Executors.newSingleThreadExecutor().asCoroutineDispatcher().use { dispatcher ->
```

Comment:
Why are we going back to explicit JDK executors?

Reply:
Reverted